### PR TITLE
feat: truncate digest post summary

### PR DESCRIPTION
**Pull request by capJavert**
Requested by [product](https://dailydotdev.slack.com/archives/C02JAUF8HJL/p1709565175570589?thread_ts=1709563527.758179&cid=C02JAUF8HJL)

### CODE REVIEW

**Review by omBratteng — APPROVED** [review]

File: src/common/personalizedDigest.ts [hunk 1]
```diff
@@ -82,7 +82,10 @@ const getPostsTemplateData = ({ posts }: { posts: TemplatePostData[] }) => {
       ),
       post_upvotes: post.upvotes || 0,
       post_comments: post.comments || 0,
-      post_summary: post.summary,
+      post_summary:
+        post.summary?.length > 150
```
Comment (omBratteng) [inline src/common/personalizedDigest.ts:86]: Only thing maybe, non blocking. Move `150` to a constant.

Comment (rebelchris) [reply src/common/personalizedDigest.ts:86]: Or even make it a prop from gb 👀👀👀👀

**Review by rebelchris — APPROVED** [review]
Non blocking comment

File: src/common/personalizedDigest.ts (hunk 1 above)
Comment (capJavert) [reply src/common/personalizedDigest.ts:86]: image

**Review by omBratteng — APPROVED** [review]

**Review by nensidosari — APPROVED** [review]
🎉

**Review by rebelchris — APPROVED** [review]
Love it! More freedom for marketing 😂

**Pull request merged**